### ISSUE TITLE
UI Hanged in Second Authentication with AllowAlternativeAuthentication on

### ISSUE THREAD

**Issue by ericc0504** [smstuebe/xamarin-fingerprint#160]
With AllowAlternativeAuthentication set to true, UI hanged in the second authentication.

## Steps to reproduce

1. Authenticate once (either set AllowAlternativeAuthentication to true or false, authenticate success or fail)

2. Authenticate again with AllowAlternativeAuthentication set to true

## Expected behavior
Open up the authentication dialog

## Actual behavior
UI Hanged

### Crashlog

N/A

## Configuration

**Version of the Plugin:** 2.0.0

**Platform:** e.g. Android 9.0

**Device:** Nokia 6.1 Plus

[BioAuthDemo.zip](https://github.com/smstuebe/xamarin-fingerprint/files/4217648/BioAuthDemo.zip)

**Comment by smsissuechecker** [comment]
Hi @ericc0504,

I'm the friendly issue checker.
Thanks for using the issue template :star2:
I appreciate it very much. I'm sure, the maintainers of this repository will answer, soon.

**Comment by smstuebe** [comment]
Damn. I had this issue with the first biometric bindings, too. But I could not reproduce it later so I thought this was fixed. 
Unfortunately I think this is an issue of the underlying androidx library. I'll try to investigate this asap.

**Comment by ericc0504** [comment]
Thank you. Look forward to the next release.

**Comment by christianaldas** [comment]
@smstuebe  Hi, bro.. Do you have resolved this Issue?

**Comment by smstuebe** [comment]
@christianaldas nope. asap != now. And as I said, I'm not sure if I can fix it, because I experienced this bug also with my own androidx bindings in the first pre releases. That's why I assume, that this bug is in the native library. I found a workaround, but can't really remember what it was (something like invalidate/reload activity). And it had the side effect, that the activity reloaded.

**Comment by christianaldas** [comment]
So, this never will be fixed?

**Comment by CSharpDevChris** [comment]
It seems that it is a problem with the Package **Xamarin.AndroidX.Biometric** Version 1.0.0. 
If I reference the Version **1.0.1** in my app the problem is not reproducable anymore.
